app/views/protectaccount/protectAccountRemovalComplete.scala.html

```diff
@@ -34,9 +34,9 @@
         <h1 class="heading-xlarge">2-step verification removed</h1>
     </div>
     <br>
-    <p>You have removed 2-step verification from your account</p>
+    <p>You can <a href="@routes.ProtectAccount.getQrCode()">add 2-step verification</a> back again at any time.</p>
 
     <div>
-        <a id="view-all-apps" class="button" type="submit" href="@routes.ManageApplications.manageApps">View all applications</a>
+        <a id="view-all-apps" class="button" type="submit" href="@routes.ProtectAccount.protectAccount()">View all applications</a>
```

Comment:
This link looks like it's going to the protect account page but it says "View all applications"